Update the script: update checkpw command and make it to work with python3

j-rivero opened the pull request:
It was failing on my Xenial system with: ```AttributeError: 'module' object has no attribute 'checkpw'```

Commit:
Update the script: update checkpw command and make it to work with python3

gavanderhoorn commented on jenkins_bcryptpw.py:
```diff
@@ -30,12 +30,12 @@
 
-print ("plaintext : {}".format(plaintext))
-print ("hash      : {}".format(pw))
-print ("check     : {}".format("passed" if bcrypt.checkpw(plaintext, pw) else "failed"))
```
`checkpw(..)` is supported from `bcrypt` version [3.1.0](https://pypi.python.org/pypi/bcrypt/3.1.0). Can you check which version you have?

j-rivero replied:
ouch, Xenial 2.0.0

```
jrivero@nium ~ $ rmadison python3-bcrypt
 python3-bcrypt | 0.4-1ubuntu2  | trusty/universe | amd64, arm64, armhf, i386, powerpc, ppc64el
 python3-bcrypt | 2.0.0-2build1 | xenial/universe | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 python3-bcrypt | 3.1.3-1       | artful/universe | amd64, arm64, armhf, i386, ppc64el, s390x
 python3-bcrypt | 3.1.4-2       | bionic/universe | amd64, arm64, armhf, i386, ppc64el, s390x
```

gavanderhoorn committed:
Just disable pw check for now.

At least till bcrypt supports it on all 'target platforms' (as much as we
have those).

gavanderhoorn commented:
Thanks for the PR.

I've just disabled the `pwcheck(..)` bit, as just running `hashpw(..)` instead doesn't make much sense to me.
